pkg/apis/componentconfig/types.go

```diff
@@ -18,419 +18,25 @@ package componentconfig
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	apiserverconfig "k8s.io/apiserver/pkg/apis/config"
+	kebectrlmgrconfig "k8s.io/kubernetes/pkg/controller/apis/config"
 )
 
-type GroupResource struct {
-	// group is the group portion of the GroupResource.
-	Group string
-	// resource is the resource portion of the GroupResource.
-	Resource string
-}
-
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-type KubeControllerManagerConfiguration struct {
+type CloudControllerManagerConfiguration struct {
```

Comment:
where will this go?

Reply:
we will move it to `cmd/cloud-controller-manager` in next PR